---
layout: page
title: Join us
order: 3
---

### Join us

If you are interested in learning more about the group, we invite you to get in touch. While prior experience with computational biology is a plus, the most important requirement is motivation!

#### Master's thesis topic available!

The available project focuses on developing computational single cell methods for chronobiology applications in collaboration with the [Kramer lab](https://www.achim-kramer-lab.de/), also at Charité. Chronobiology and -medicine revolve around biological rhythms and their significance in diagnostic and therapeutic applications. Here, we are looking to predict circadian phase for individual cells based on single-cell RNAseq data - a task which is challenging due to sparsity and noise typical of such data - using state of the art single cell analysis methods. A previous master’s student has generated first promising results in this direction which we are looking to expand on for publication.
 
Candidates must be available to work in the lab full time for at least 6 months. The desired start date ist between December 2024 and March 2025. Applicants who have demonstrated prior computational project experience (i.e. at least one computational lab rotation project or equivalent) are welcome to reach out to me via mail at lisa.buchauer@charite.de with a short paragraph describing their motivation and their CV and transcript of records attached.

#### Bachelor's theses, Master's theses, MD theses, lab rotation projects, internships
If you are interested in doing any of the above in my lab, I will be happy to hear from you to discuss your interests and currently available project options. Please reach out to me __with your CV and transcript of records__ at [lisa.buchauer@charite.de](mailto:lisa.buchauer@charite.de). 

#### PhD & PostDoc

There are currently no explicit positions to be filled, but qualified candidates interested in PhD or PostDoc opportunities are welcome to reach out to discuss potential funding options.  

We are interested in candidates from quantitative backgrounds (physics, mathematics, computer science, computational biology, bioinformatics or similar) with a desire to apply their skills to biomedical research questions. Good command of at least one programming language (e.g. python, R, Julia) is required. Candidates with life science background (biology, biomedicine, biochemistry or similar) are also welcome to apply if they have a strong analytical side and the desire to expand their theoretical and computational skills.  

Prior experience with any of the following is considered a plus: omics data analysis, mathematical modeling, machine learning, spatial or temporal data analysis, software development principles and biological wet-lab work. Candidates should be curious, willing to work in a multidisciplinary team, and have a strong sense of ownership for their projects. Fluency in English is required.





