---
layout: page
permalink: /MD24/
---

#### MD / Dr. med. project available: "Computational Analysis of Circadian Heterogeneity in Liver Pathologies Using Single-Cell Transcriptomics Data"

We are looking for a Charité medical student to join us for their MD thesis. The work will include identifying relevant datasets in the literature, bioinformatic processing of such datasets, applying and improving single-cell analysis methods and interpreting differential gene expression results. A start date in Q1 or Q2 2025 is desired. We will aim for a doctorate by publication.

#### Background
The circadian rhythm plays a fundamental role in regulating physiological processes. The fields of chronobiology and chronomedicine study these biological rhythms, their disruptions, and links to pathologies. The liver exemplifies an organ with a tightly regulated circadian clock, and its disruption is linked to various pathologies including metabolic diseases and cancer. Recent advances in single-cell transcriptomics now enable examination of circadian patterns at the cellular level, offering new insights into how these patterns change in liver diseases.
In this project, we will build on our lab's previous work in assigning circadian phase estimates to individual cells based on single-cell transcriptomics data. We will further develop these computational methods and apply them to publicly available single-cell transcriptomics datasets of healthy and diseased liver, studying the interplay between circadian rhythms, cellular synchronization, and liver pathologies. The project will be conducted in collaboration with chronobiologist Prof. Dr. Achim Kramer (Charité) and liver systems biologist Dr. Rita Manco (UC Louvain).

#### Aims
1.	Apply and further develop existing computational methods for determining circadian phase from publicly available single-cell transcriptomics data of the liver
2.	Characterize the distribution of circadian phases across individual liver cells in healthy and diseased liver datasets; quantify and compare the degree of desynchronization across different pathologies
3.	Identify and analyze differentially expressed genes and pathways between phase-leading and phase-lagging cells within and across datasets

#### What we are looking for
-	Curious individuals with interest in quantitative / computational work in an interdisciplinary, translational context
-	Basic programming skills in python, R or similar OR commitment to complete a relevant online course (e.g. [“data analysis for life sciences”](https://www.edx.org/certificates/professional-certificate/harvardx-data-analysis-for-life-sciences) (participation is free, you do not need to pay for the certification))
-	Candidates who are at least in the 5th semester of their medical studies

#### What we offer
-	Work in an international computational team (https://libuchauer.github.io/team/) with strong ties to experimentalists and clinicians
-	Friendly, supportive environment with flexible work hours
-	Work with state-of the art computational methods for the analysis of high-dimensional biomedical data sets (e.g. omics data), access to high-power compute facilities
-	Support with scholarship applications and conference participation

#### How to apply
Please send your CV as well as a short paragraph outlining your motivation and a description of your prior experience with computational / data analysis projects (if applicable) to lisa.buchauer@charite.de. If you are interested in the project but unsure whether you are qualified, please do not self-exclude but reach out with your questions.





